src/test/groovy/edu/ie3/datamodel/io/factory/input/Transformer2WInputFactoryTest.groovy

```diff
@@ -63,4 +64,29 @@ class Transformer2WInputFactoryTest extends Specification implements FactoryTest
       assert autoTap
     }
   }
-}
\ No newline at end of file
+  def "A Transformer2WInputFactory should throw an IllegalArgumentException if nodeA is on the lower voltage side"() {
+    given: "a system participant input type factory and model data"
+    def inputFactory = new Transformer2WInputFactory()
+    Map<String, String> parameter = [
+      "uuid"           : "91ec3bcf-1777-4d38-af67-0bf7c9fa73c7",
+      "operatesfrom"   : "2019-01-01T00:00:00+01:00[Europe/Berlin]",
+      "operatesuntil"  : "",
+      "id"             : "TestID",
+      "paralleldevices": "2",
+      "tappos"         : "3",
+      "autotap"        : "true"
+    ]
+    def inputClass = Transformer2WInput
+    def operatorInput = Mock(OperatorInput)
+    def nodeInputA = GridTestData.nodeB
```

Comment:
Do we want to accept transformers with the same voltage on both sides? If yes this test need to be adapted that we then don't expect an exception. If not the method itself should check for greaterThanOrEqual. Same for three- winding transformer.

Reply:
As per our discussion we allow the equal case.